spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -1691,12 +1698,20 @@ def _do_mapping(self, run_time, n_machine_time_steps, total_run_time):
         # Create a buffer manager if there isn't one already
         if not self._use_virtual_board:
             if self._buffer_manager is None:
-                inputs["StoreBufferDataInFile"] = self._config.getboolean(
-                    "Buffers", "store_buffer_data_in_file")
+                inputs["BufferDatabaseFile"] = os.path.join(
+                    self._report_default_directory, "buffer.sqlite3")
                 algorithms.append("BufferManagerCreator")
                 outputs.append("BufferManager")
             else:
                 inputs["BufferManager"] = self._buffer_manager
+            if self._java_caller is None:
+                if self._config.getboolean("Java", "use_java"):
+                    java_call = self._config.get("Java", "java_call")
+                    javaspinnaker_path = self._config.get_str(
+                        "Java", "javaspinnaker_path")
```

Comment:
```suggestion
                        "Java", "java_spinnaker_path")
```

Reply:
if changing that makes you happy...

Reply:
it does. it makes it read easier. and stops pycharm complaining about typos